### ISSUE TITLE
Page ID NN is not routable

### ISSUE THREAD

**Issue by e-spin** [terminal42/contao-folderpage#37]
When I debug the routing the following error message appears - this is a "folder" type page.

C 4.13, FP 3.1.0

```
php vendor/bin/contao-console debug:router
13:00:34 CRITICAL  [console] Error thrown while running command "debug:router". Message: "Page ID 105 is not routable" ["exception" => Symfony\Component\Routing\Exception\ResourceNotFoundException^ { …},"command" => "debug:router","message" => "Page ID 105 is not routable"]

In FolderPageController.php line 38:
                               
  Page ID 105 is not routable  
                               
```

**Comment by fritzmg** [comment]
I can confirm the issue. This happens with any folder page that is either hidden in the menu or has no published children:

```
$ vendor/bin/contao-console debug:router -v

20:53:48 CRITICAL  [console] Error thrown while running command "debug:router -v". Message: "Page ID 61 is not routable" ["exception" => Symfony\Component\Routing\Exception\ResourceNotFoundException^ { …},"command" => "debug:router -v","message" => "Page ID 61 is not routable"]

In FolderPageController.php line 38:

  [Symfony\Component\Routing\Exception\ResourceNotFoundException]
  Page ID 61 is not routable


Exception trace:
  at vendor\terminal42\contao-folderpage\src\Controller\FolderPageController.php:38
 Terminal42\FolderpageBundle\Controller\FolderPageController->configurePageRoute() at vendor\contao\core-bundle\src\Routing\Page\PageRegistry.php:84
 Contao\CoreBundle\Routing\Page\PageRegistry->getRoute() at vendor\contao\core-bundle\src\Routing\RouteProvider.php:182
 Contao\CoreBundle\Routing\RouteProvider->addRoutesForPage() at vendor\contao\core-bundle\src\Routing\RouteProvider.php:142
 Contao\CoreBundle\Routing\RouteProvider->addRoutesForPages() at vendor\contao\core-bundle\src\Routing\RouteProvider.php:130
 Contao\CoreBundle\Routing\RouteProvider->getRoutesByNames() at vendor\symfony-cmf\routing\src\LazyRouteCollection.php:57
 Symfony\Cmf\Component\Routing\LazyRouteCollection->all() at vendor\symfony\routing\RouteCollection.php:173
 Symfony\Component\Routing\RouteCollection->addCollection() at vendor\symfony-cmf\routing\src\ChainRouteCollection.php:88
 Symfony\Cmf\Component\Routing\ChainRouteCollection->all() at vendor\symfony\framework-bundle\Console\Descriptor\TextDescriptor.php:57
 Symfony\Bundle\FrameworkBundle\Console\Descriptor\TextDescriptor->describeRouteCollection() at vendor\symfony\framework-bundle\Console\Descriptor\Descriptor.php:47
 Symfony\Bundle\FrameworkBundle\Console\Descriptor\Descriptor->describe() at vendor\symfony\console\Helper\DescriptorHelper.php:65
 Symfony\Component\Console\Helper\DescriptorHelper->describe() at vendor\symfony\framework-bundle\Command\RouterDebugCommand.php:127
 Symfony\Bundle\FrameworkBundle\Command\RouterDebugCommand->execute() at vendor\symfony\console\Command\Command.php:298
 Symfony\Component\Console\Command\Command->run() at vendor\symfony\console\Application.php:1058
 Symfony\Component\Console\Application->doRunCommand() at vendor\symfony\framework-bundle\Console\Application.php:96
 Symfony\Bundle\FrameworkBundle\Console\Application->doRunCommand() at vendor\symfony\console\Application.php:301
 Symfony\Component\Console\Application->doRun() at vendor\symfony\framework-bundle\Console\Application.php:82
 Symfony\Bundle\FrameworkBundle\Console\Application->doRun() at vendor\symfony\console\Application.php:171 Symfony\Component\Console\Application->run() at vendor\contao\manager-bundle\bin\contao-console:38
 include() at vendor\bin\contao-console:119
```

**Comment by aschempp** [comment]
should be fixed by 17f50a2da2d90f43c9c14ecf872ce4c5f5920dac
